### ISSUE TITLE
Make sentence text itself the click target

### ISSUE THREAD

**Issue by githubshrek** [Tatoeba/tatoeba2#2779]
Instead of a coloured arrow to jump to the sentence of interest (in the OLD design):
![image](https://user-images.githubusercontent.com/7943077/117542978-1a7df500-b01b-11eb-9019-9c56a089e464.png)

Instead of an "info" icon (in the NEW design):
![image](https://user-images.githubusercontent.com/7943077/117543020-47320c80-b01b-11eb-8589-eda2ec2cb2c9.png)

Instead of these indirect ways of linking to a sentence, the displayed text should itself be the click target. For example in the above examples, "Tom cut the cake" and "She told me she was glad" (and of course all the other sentences displayed) should be clickable.

Why? Usability. You have an extremely obvious and intuitive click target (the sentence text itself), and instead of making it clickable, currently an extra click target (the arrow or the info icon) has been added. If you can imagine Wikipedia having every link using a coloured arrow or an info icon next to each link, you can see how far from ideal this is.

You can get an idea of how intuitive this would be from my own site Lingopolo, which is also basically a collection of words and phrases, but in the case of Lingopolo used for teaching languages, see for example:

- https://lingopolo.org/french/recent-additions
- https://lingopolo.org/french/dictionary/english-french/a
- https://lingopolo.org/french/word/dog

A few details:

- I would not have the page of the sentence itself link back to itself. For example, on https://lingopolo.org/french/word/they-have-a-dog the sentence itself is not a link, but all the other words/sentences referenced are links
- the link on the text needs to be clearly indicated using the normal visual cues e.g. mouse pointer changing shape, colour of the links being different from the text which is not clickable, hover of mouse causing a sentence underline or other visual cue that you can click)
- I would get rid of the "info" icon, since it would no longer be needed

**Comment by alanfgh** [comment]
I very much like the idea of making the text the link and getting rid of the "i" icon. Eliminating that column would provide some welcome additional space (which could be "reinvested" by making some other column wider) and remove the need for having to move the mouse as far. Often, in situations like this, making text into a link makes the text harder to copy, but here that's not an issue because there's an icon entirely devoted to copying the text of the sentence.

**Comment by jiru** [comment]
Yeah this would be a welcomed UI improvement but we could very well frustrate users who are just trying to select the text, for example copy-paste a single word onto a dictionary by double-clicking. As Alan pointed out, the situation is now a bit different because of the "copy text"; still, it is not clear to me what would be the best UI here. See also past discussions in #2103 for more insights.

> visual cues e.g. mouse pointer changing shape, colour of the links being different from the text which is not clickable, hover of mouse causing a sentence underline or other visual cue that you can click)

Note that this does not happen on touch screens. But we could as well provide a slightly different UI for mobile to get around the problem.

**Comment by mramosch** [comment]
I am e.g. using the text to trigger Text-To-Speech for single words or phrase selection.

So I’d rather prefer the current info button to open a sentence page...

**Comment by githubshrek** [comment]
> I am e.g. using the text to trigger Text-To-Speech for single words or phrase selection.
> 
> So I’d rather prefer the current info button to open a sentence page and leave the functionality of the text as is...



I understand text-to-speech of course, but there are a few things I would say:

- what makes it so that you can do text-to-speech if the text is plain text but you can't do text-to-speech if the text is part of a link? Please give very clear details so that I can understand your use case better.
- every sentence will not be linked in its own sentence page, only on the other pages such as the list views, or the translations of the current sentence
- I _think_ the huge UI improvement for the large majority of users would likely override the inconvenience for the few that might have a very specific use-case
